dcicutils/qa_utils.py

```diff
@@ -4,13 +4,27 @@
 
 import contextlib
 import datetime
+import time
 import io
 import os
 import pytz
 
 from .misc_utils import PRINT, ignored, Retry
 
 
+@contextlib.contextmanager
+def timed():
+    """ A simple context manager that will time how long it spends in context. Useful for debugging. """
+    start = time.time()
+    try:
+        yield
+    except Exception:
```

Comment:
Is this doing something? It seems like a no-op. Maybe you sometimes put code in the middle, in which case I'd write this (using `dcicutils.misc_utils.ignored` to keep tools from fussing about sometimes unused `e`...
```
except Exception as e:
    ignored(e)
    # Debugging code might want to go here.
    raise
```